sunpy/map/sources/tests/test_xrt_source.py

```diff
@@ -38,6 +38,11 @@ def test_measurement(xrt_map):
     assert xrt_map.measurement == 'Be thin-Open'
 
 
+def unit(xrt_map):
```

Comment:
```suggestion
def test_unit(xrt_map):
```

Reply:
Without the `test_` prefix on the name, I don't think pytest will run this so I don't think this test has actually been run yet...

Reply:
yes, good catch! :cowboy_hat_face: